src/test/java/com/stripe/functional/ChargeTest.java

```diff
@@ -25,6 +25,17 @@ public void testChargeCreate() throws StripeException {
     }
 
     @Test
+    public void testChargeCreateExpandBT() throws StripeException {
```

Comment:
It looks like this test is named with "BT" while another new one down below uses "BalanceTransaction". Can we just use the full name for consistency? I think it's fine if locals still contain "BT".